spark/agent/src/main/scala/za/co/absa/spline/harvester/builder/write/WriteCommandExtractor.scala

```diff
@@ -98,6 +107,18 @@ class WriteCommandExtractor(pathQualifier: PathQualifier, session: SparkSession)
     val sourceIdentifier = SourceIdentifier.forTable(table)(pathQualifier, session)
     WriteCommand(name, sourceIdentifier, mode, query, Map("table" -> table))
   }
+
+  private val toBeImplemented = Seq(classOf[AlterTableAddColumnsCommand], classOf[AlterTableChangeColumnCommand],
+    classOf[AlterTableRenameCommand], classOf[AlterTableSetLocationCommand], classOf[CreateDataSourceTableCommand],
+    classOf[CreateDatabaseCommand], classOf[CreateTableLikeCommand], classOf[DropDatabaseCommand],
+    classOf[LoadDataCommand], classOf[TruncateTableCommand])
+
+  private def logUnimplementedComamnd(c: LogicalPlan): Unit = {
```

Comment:
What if we vary the behavior of this method depending on the `SplineMode` property?
So that in the _BestEffort_ mode we'd log a warning, but in the _Required_ mode we throw an exception instead.